packages/main/src/TableSelection.ts

```diff
@@ -26,7 +26,7 @@ import { isSelectionCheckbox, isHeaderSelector, findRowInPath } from "./TableUti
  * * Multiple - select multiple rows.
  * * None - no selection active.
  *
- * As the selection is key-based, `ui5-table-row` components need to define a unique `key` property.
+ * As the selection is key-based, `ui5-table-row` components need to define a unique `row-key` property.
```

Comment:
like F6 handling we should document the TableSelection import. better in the "features" slot documentation of the table as general suggestion and here in the TableSelection.ts

Reply:
it would be great to mention the import again in the documentation of the TableSelection